source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
+of your code.  See the :doc:`Builders overview page
+</fundamentals/builders/>` for a list of Builders available in the Java
+driver. 
+
+
+Filters are the operations MongoDB uses to limit your results to what
+you want to see. Some examples of results from queries with filters are: 
+
+- Items that cost $0 to $25
+- A hotel with amenities that include an indoor swimming pool and free parking
+- A food critic review that mentions "spicy" 
+
+This guide shows you how to use builders with examples of the following
+types of operators: 
+
+- :ref:`Comparison <comparison>`
+- :ref:`Logical <logical>`
+- :ref:`Arrays <arrays>`
+- :ref:`Elements <elements>`
+- :ref:`Evaluation <evaluation>`
+- :ref:`Bitwise <bitwise>`
+- :ref:`Geospatial <geospatial>`
+
+The :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+class provides static factory methods for all the MongoDB query
+operators. Each method returns an instance of the :ref:`Bson <bson>`
+type, which you can pass to any method that expects a query filter. 
+
+.. tip::
+
+   For brevity, you may choose to import all methods of the
+   :java-core-api:`Filters <com/mongodb/client/model/Filters.html>`
+   class statically: 
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Filters.*;
+
+   The examples below assume this static import.
+
+The following sections show you how to use the ``Filters`` class to sort
+your queries using the following setup:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin imports
+   :end-before: end imports 
+
+The Filter examples in this guide use the following sample collections:
+
+Collection: ``paint_purchases``
+
+.. code-block:: json
+
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+   {"_id": 4, "color": "white", "qty": 6, "vendor": ["D"]}
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+   {"_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"]}
+   {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+Collection: ``binary_numbers``
+
+.. code-block:: json
+
+   { _id: 9, a: 54, binaryValue: "00110110" }
+   { _id: 10, a: 20, binaryValue: "00010100" }
+   { _id: 11, a: 68, binaryValue: "1000100" }
+   { _id: 12, a: 102, binaryValue: "01100110" }
+
+Collection: ``geo_points``
+
+.. code-block:: json
+
+   { _id: 13, "coordinates": { type: "Point", coordinates: [2.0, 2.0] } }
+   { _id: 14, "coordinates": { type: "Point", coordinates: [5.0, 6.0] } }
+   { _id: 15, "coordinates": { type: "Point", coordinates: [1.0, 3.0] } }
+   { _id: 16, "coordinates": { type: "Point", coordinates: [4.0, 7.0] } }
+
+.. _comparison:
+
+Comparison 
+----------
+
+The comparison operator compares the value in a document to a specified value.
```

Comment:
I think there are two different ways to present each of the operator types that might be better than this. In the current form, it sounds like there is a single "comparison operator", but it is actually a category of several operators which you show in the table below.
One way to resolve this is to refer to the operators in plural form. E.g. "The comparison operators compare..."
Or to identify it as a category. E.g. "The comparison filters include all operators that compare ..."